### PR TITLE
fix: update rbdms urls to new repos

### PR DESCRIPTION
**Pull request by moT01**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/chinese-traditional/13-relational-databases/learn-relational-databases/learn-github-by-building-a-list-of-inspirational-quotes.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ id: 602da04222201c65d2a019f3
 title: Learn GitHub by Building a List of Inspirational Quotes
 challengeType: 12
 helpCategory: Relational Databases
-url: https://github.com/moT01/.learn-github-by-building-a-list-of-inspirational-quotes
+url: https://github.com/freeCodeCamp/learn-github-by-building-a-list-of-inspirational-quotes
```
Comment (gikf) [inline curriculum/challenges/chinese-traditional/13-relational-databases/learn-relational-databases/learn-github-by-building-a-list-of-inspirational-quotes.md:6]: Just to point out - this link leads to 404 error

Comment (moT01) [reply curriculum/challenges/chinese-traditional/13-relational-databases/learn-relational-databases/learn-github-by-building-a-list-of-inspirational-quotes.md:6]: thanks @gikf - yea, I probably could have left that one for now.

**Review by scissorsneedfoodtoo — APPROVED** [review]
Everything LGTM. Tested a couple of challenges locally and they load up in CodeAlly just fine.

@moT01, real quick -- the [Inspirational Quotes](https://github.com/moT01/learn-github-by-building-a-list-of-inspirational-quotes) URL still 404s after the last change.

Do you want to revert back to the hidden folder style repo name? Or do you plan to change the repo name to match the URL in this PR?

**Comment by moT01** [comment]
I'll just change the repo name

**Review by gikf — APPROVED** [review]
LGTM :+1:

**Pull request merged**